services/src/main/java/org/exoplatform/webconferencing/server/UpdatePollingServlet.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (C) 2003-2017 eXo Platform SAS.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.exoplatform.webconferencing.server;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import javax.servlet.AsyncContext;
+import javax.servlet.AsyncEvent;
+import javax.servlet.AsyncListener;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.exoplatform.container.web.AbstractHttpServlet;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.webconferencing.UserCallListener;
+import org.exoplatform.webconferencing.WebConferencingService;
+
+/**
+ * Created by The eXo Platform SAS.
+ *
+ * @author <a href="mailto:pnedonosko@exoplatform.com">Peter Nedonosko</a>
+ * @version $Id: UpdatePollingServlet.java 00000 Jul 18, 2017 pnedonosko $
+ */
+public class UpdatePollingServlet extends AbstractHttpServlet {
```

Comment:
Is it a fallback for Cometd ? If so, why do we need it ? Cometd should handle fallback directly.

Reply:
Historically, this long-polling servlet, was a first solution for pushing data from the server (incoming call notification). Should be removed in GA.